From 5f0e537c89d9f3b356166c409303b38777f2e3de Mon Sep 17 00:00:00 2001
From: Lorenzo Dal Col <lorenzo@khronosgroup.org>
Date: Thu, 18 Jan 2024 22:13:49 +0100
Subject: [PATCH] Change zlib URL because the one from zlib.net requires a
 human-verification

VK-GL-CTS issue: 4895

This does not affect any test.

Change-Id: I7002062f46f957dd865d66387dde3e53d3b2c5f0
Signed-off-by: Eric Engestrom <eric@igalia.com>
---
 external/fetch_sources.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/external/fetch_sources.py b/external/fetch_sources.py
index 081aea023d5683698653..7b107d5026b3dbb246c2 100644
--- a/external/fetch_sources.py
+++ b/external/fetch_sources.py
@@ -290,7 +290,7 @@ def postExtractLibpng (path):
 
 PACKAGES = [
 	SourcePackage(
-		"http://zlib.net/fossils/zlib-1.2.13.tar.gz",
+		"https://github.com/madler/zlib/releases/download/v1.2.13/zlib-1.2.13.tar.gz",
 		"b3a24de97a8fdbc835b9833169501030b8977031bcb54b3b3ac13740f846ab30",
 		"zlib"),
 	SourcePackage(
-- 
Cheers,
  Eric

